inputs/orchestrator_inner:6.json

```diff
@@ -56,6 +56,9 @@
     },
     {
       "name": "resolve_remote_source"
+    },
+    {
+      "name": "pin_operator_digest"
```

Comment:
Had no idea where in the sequence this should run, I just added it at the end

Reply:
IMHO this location is fine